paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/FlowControllerTest.kt

```diff
@@ -1012,7 +1012,7 @@ internal class FlowControllerTest {
             method("GET"),
             path("/v1/elements/sessions"),
         ) { response ->
-            response.setResponseCode(400)
+            response.setResponseCode(500)
```

Comment:
I still think we should also add a test for connection failure -- in addition to a 500. See https://github.com/stripe/stripe-android/blob/1b20a1e12f93a2b362d4e14db8255e2e3f0cc80a/paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/PaymentSheetTest.kt#L193 as an example.